.circleci/config.yml

```diff
@@ -2,24 +2,65 @@ version: 2.1
 orbs:
   cypress: cypress-io/cypress@1
   codecov: codecov/codecov@1.0.4
+  aws-s3: circleci/aws-s3@1.0.11
+
 jobs:
+  build:
+    docker: 
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - npm-{{ checksum "package-lock.json" }}
+            - npm-
+      - run:
+          name: Set registry URL
+          command: 'npm config set "@fortawesome:registry" https://npm.fontawesome.com/'
+      - run:
+          name: Authenticate with registry
+          command: 'npm config set "//npm.fontawesome.com/:_authToken" $FONTAWESOME'
+      - run:
+          name: Install
+          command: npm i --no-dev
+      - run: 
+          name: Run build script
+          command: 'npm run build'
+      - save_cache:
+          key: npm-{{ checksum "composer.lock" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Create built_code directory
+          command: mkdir -p ./built_code/
+      - run:
+          name: move dist directory into built_code directory
+          command: mv ./dist built_code/
+      - persist_to_workspace:
+          root: ./built_code
+          paths:
+            - dist
   deploy:
-    executor: cypress/base-10
+    docker:
+      - image: 'circleci/python:2.7'
```

Comment:
I don't think this orb requires python, it only lists bash.. is there any reason not to use the previous environment?  It'll be a lot faster as you won't have to run npm install or deal with using caches

Reply:
AWS CLI needs python.

Reply:
See note about aws-s3 orb not having jobs defined, but only commands below.